incubator/vault/values.yaml

```diff
@@ -6,14 +6,24 @@ image:
   repository: vault
   tag: 0.9.0
   pullPolicy: IfNotPresent
+
+deployment:
+  ## Deployment annotations
+  annotations: {}
+  #   name: value
+  replicas:
```

Comment:
I don't like `replicas`. It is common to called them `podAnnotations`. Could you change that? The nesting under deployment is not ideal as it is now because anything else that's not under ingress or service would belong there as well. Maybe just remove this nesting and go with `annotations` and `podAnnotations`.